src/olympia/reviewers/templates/reviewers/moderationlog.html

```diff
@@ -1,5 +1,7 @@
 {% extends "reviewers/base.html" %}
 
+{% block title %} Reviewer Moderation Log – Add-ons for Firefox {% endblock %}
```

Comment:
And the rest of these strings similarly wrapped to be localizable.  

(in jinja2 templates `{{ _('...') }}` is generally used for single lines; `{% trans %}...{% endtrans %}` is used for multiple lines)